packages/ui/constants/index.ts

```diff
@@ -58,3 +58,468 @@ export const MINUTES_PER_YEAR = 24 * 365 * 60;
 
 export const UP_LIMIT = 0.005;
 export const DOWN_LIMIT = 0;
+
+// for additional APR for aBNBC in Ankr
+export const aBNBcContractAddress = '0xBb1Aa6e59E5163D8722a122cd66EBA614b59df0d';
```

Comment:
I believe this address is also availabe in the sdk, in the `assets` for tokenSymbol: `aBNBc`

Reply:
This is not a token address for `aBNBc`

Reply:
you're right, ignore